Mathlib/Tactic/ToAdditive/Frontend.lean

```diff
@@ -93,6 +75,27 @@ the fifth argument before the third argument. This is used in `to_dual` to swap
 -/
 syntax toAdditiveReorderOption := &"reorder" " := " (num+),+
 /--
+the `relevant_arg := ...` option tells which argument is a type where this declaration uses the
+multiplicative structure. This is inferred automatically using the function `findMultiplicativeArg`,
+but it can also be overwritten using this syntax.
+
+If there are multiple arguments with a multiplicative structure, we typically tag the first one.
+If this argument contains a fixed type, this declaration will not be additivized.
+See the Heuristics section of the `to_additive` doc-string for more details.
+
+If a declaration is not tagged, it is presumed that the first argument is relevant.
+
+To indicate that there is no relevant argument, set it to a number that is out of bounds,
+i.e. larger than the number of arguments, e.g. `(relevant_arg := 100)`
```

Comment:
Shoudl this trick be mentioned in the `to_additive` docstring as well? Maybe it doesn't come up that frequently.